src/indicator/algorithm/calculator/undefinedInputAdapter.js

```diff
@@ -1,54 +0,0 @@
-import d3 from 'd3';
-import slidingWindow from './slidingWindow';
-
-// Indicator algorithms are not designed to accomodate leading 'undefined' value.
-// This adapter adds that functionality by adding a corresponding number
-// of 'undefined' values to the output.
-export default function() {
```

Comment:
Sounds good to me - I think there's more to discuss here given https://github.com/d3fc/d3fc-technical-indicator/issues/10 - but perhaps that's about configuration rather than consistency of the shape of the output?